setup.py

```diff
@@ -41,6 +41,7 @@
         'passlib',
         'shortuuid',
         'webgrid',
+        'webhelpers2',
```

Comment:
Thanks for adding this for now, since it's now missing from webgrid. We'll need to do separate maintenance to get this out of here again.